src/store.rs

```diff
@@ -196,6 +217,23 @@ impl Storage for RocksDB {
             .put(key.encode(), encode(&value))
             .map_err(|_| StorageError::WriteError)
     }
+
+    fn get_all_keys(&self) -> Vec<StorageKey> {
+        let mut iter = self.db.raw_iterator();
+        iter.seek_to_first();
+        let mut keys = Vec::new();
+        while iter.valid() {
+            let key = iter.key().unwrap();
+            let address = H160::from_slice(&key[..20]);
+            let value = U256::from_big_endian(&key[20..]);
+            keys.push(StorageKey::new(address, value));
+            iter.next();
+        }
+        keys
+    }
+    fn fake_clone(&self) -> InMemory {
+        unimplemented!()
+    }
```

Comment:
For the `RocksDB` logic we may just want to open a [transaction](https://github.com/facebook/rocksdb/wiki/Transactions).